Automattic/jetpack
# Jetpack VideoPress: update logos

This PR updates the logos in the VideoPress plugin.

Fixes 1202858161751496-as-1203427126857118

#### Changes proposed in this Pull Request:
- Added the `JetpackVideoPressLogo` component to the `jetpack-components` package.
- Updated the logos in `package/videopress`.
- Removed the `Logo` component from `package/videopress`.

#### Other information:


#### Jetpack product discussion
Context: p1HpG7-jkl-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:

- Spin up a test site
- Build and install the Jetpack VideoPress plugin
- Visit the `Jetpack > VideoPress` section
- Note the logo is updated (see captures below)
- Purchase a subscription to VideoPress
- Notice that the logo is updated (see captures below)
- Upload a video, and go to the edit section
- Check that the logo is updated (see captures below)

#### Captures

_Price page_
| Before | After |
|--------|------|
|<img width="500" alt="Screen Shot 2022-12-07 at 11 10 10 AM" src="https://user-images.githubusercontent.com/1620183/206236029-65737e21-00fd-44fa-bf47-38da0c0b5a98.png">|<img width="500" alt="Screen Shot 2022-12-07 at 11 09 27 AM" src="https://user-images.githubusercontent.com/1620183/206236041-2726f143-e354-449c-b419-474bb6d7622c.png">|

_Home page_
| Before | After |
|--------|------|
|<img width="500" alt="Screen Shot 2022-12-07 at 10 57 33 AM" src="https://user-images.githubusercontent.com/1620183/206235964-183c2faf-5bd5-40a0-9178-19cbfeeb1124.png">|<img width="500" alt="Screen Shot 2022-12-07 at 11 07 37 AM" src="https://user-images.githubusercontent.com/1620183/206235970-009d58c5-dcef-4bd1-b60f-ae92fc69e8c9.png">|

_Edit page_
| Before | After |
|--------|------|
|<img width="500" alt="Screen Shot 2022-12-07 at 11 16 38 AM" src="https://user-images.githubusercontent.com/1620183/206236103-efe554a2-fe40-40a4-8a97-8037c670a3ed.png">|<img width="500" alt="Screen Shot 2022-12-07 at 11 25 36 AM" src="https://user-images.githubusercontent.com/1620183/206236107-66f2c5c7-236a-47d7-847b-d14eda7ec70c.png">|
